w2c2/c.c

```diff
@@ -1051,15 +1068,17 @@ wasmCWriteLiteral(
         case wasmValueTypeF64: {
             U64 bits = (U64) value.i64;
             if ((bits & 0x7ff0000000000000ULL) == 0x7ff0000000000000ULL) {
-                const char* sign = (bits & 0x8000000000000000ULL) ? "-" : "";
+                bool is_negative = (bits & 0x8000000000000000ULL);
```

Comment:
This is causing the `float_literals` `f64.negative_infinity` test case to fail.

Fix (and follow naming scheme):

```suggestion
                bool isNegative = (bits & 0x8000000000000000ULL) != 0;
```

Reply:
As above (with 64 bits): https://godbolt.org/z/5qGYz76eh